src/components/SlateEditor/PlainTextEditor.tsx

```diff
@@ -70,8 +64,8 @@ const PlainTextEditor: React.FC<Props> = props => {
         });
       }}>
       <Editable
-        onBlur={(event: React.FocusEvent<HTMLDivElement>) => onBlur(event, editor)}
-        onKeyDown={editor.onKeyDown}
+        onBlur={(event: FocusEvent<HTMLDivElement>) => onBlur(event, editor)}
+        onKeyDown={(editor.onKeyDown as unknown) as React.KeyboardEventHandler<HTMLDivElement>}
```

Comment:
Det er forskjell fra denne fila og RichTextEditor for onKeyDown. Har det nokke å sei?

Reply:
Forskjellig måter å omgå problemet med at Event-typene ikke matcher. Burde finne en bedre løsning på sikt, men det fungerer bra. Har endret PlainTextEditor nå til å bruke ts-ignore heller